CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
 
+## [Zebra 1.0.1](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.1) - 2023-XX-XX
+
+### Breaking Changes
+
+- Zebra now requires an argument name before the list of tracing filters when running `start` without an explicit subcommand.
```

Comment:
It might help to describe the impact of this change on most users, then provide an example of the workaround a few users might need.

```suggestion
- Zebra now detects subcommand name typos on the command-line. If you want to give Zebra a list of tracing filters, use `zebrad start --filters debug,...` ([#7056](https://github.com/ZcashFoundation/zebra/pull/7056))
```

Reply:
Thank you!